src/public/app/library/library.service.ts

```diff
@@ -1,14 +1,15 @@
 import { Http } from 'angular2/http';
 import { Injectable } from 'angular2/core';
 import { Observable } from 'rxjs/Observable';
-
 import 'rxjs/add/operator/map';
+
 @Injectable()
-export default class LibraryService {
-      constructor(public http: Http) {}
+class LibraryService {
+    constructor(public http: Http) {}
 
-      public getLibrary(): Observable<any> {
-        // noinspection TypeScriptUnresolvedFunction
-            return this.http.get('/api/').map((songs: any) => songs.json());
-      }
+    public getLibrary(): Observable<any> {
```

Comment:
Can you get rid of the "any"s here? What type of Observable does this return? Also what's the type of songs that comes back?